src/test/java/com/synopsys/integration/jenkins/scan/service/BridgeDownloadParameterServiceTest.java

```diff
@@ -107,4 +112,9 @@ void getBridgeDownloadParamsNullTest() {
         assertNotNull(result.getBridgeDownloadVersion());
         assertNotNull(result.getBridgeInstallationPath());
     }
+
+
+    public String getHomeDirectory() {
```

Comment:
Need to check if can we move this method in `Utility` class since we are using this in many places.

Reply:
As this method is only used within the test class, that's why I didn't move it to **Utility** class.